src/main/scala/org/neo4j/spark/dataframe/Neo4jDataFrame.scala

```diff
@@ -265,7 +199,7 @@ object CypherTypes {
     case "BOOLEAN" => BOOLEAN
     case "BOOL" => BOOLEAN
     case "DATETIME" => DATETIME
-	case "TIME" => TIME
+    case "TIME" => DATETIME
```

Comment:
Are you sure?

Reply:
TIME -> DATETIME?

Reply:
Just because otherwise this will not work:

https://github.com/neo4j-contrib/neo4j-spark-connector/blob/70ae1f647b6f66eddee666e55d048faded17f5f0/src/main/scala/org/neo4j/spark/Executor.scala#L24